### ISSUE TITLE
Every app crashes with the driver//used to work fine

### ISSUE THREAD

**Issue by ddylard** [jkevin/PS3EyeDirectShow#15]
I had this weird issue where everything was working fine, ps3 camera obs zoom etc.

The other day i had a weird issue regarding a sudden usb problem. I have a usb hub and both peripherals connected to it started malfunctioning (disconnect and reconnect) (mouse and Bluetooth adapter). After switching all my usb to different ports i managed to get everything working again perfectly, except  my ps3 camera.
It makes OBS crash when it looks for the directshowfilter on startup, even after i unistall the driver. The only solution to stop obs crashing is to delete ps3driver folder on program files.

It also makes chrome and vlc crash as soon as i try to test the camera. I have tried uninstalling the driver, the camera rebooting SEVERAL times. Then i tried installing forst the driver and then connecting the camera. Nothing is working.

I must clarify the camera works ok on a different pc (laptop) and on my PC with the CL driver but i dont like that driver and it doesnt work on OBS.

I cant try ps3eye driver on laptop because the installer say that the dualcore processor is not supported.

I know its a driver issue and now i have this 2 unknown devices showing up with errors on my control panel and i uninstall them but they keep reappearing every boot and i dont know if they are related to the camera because they show up regardless the camera is connected or not.

I need help guys. I've been using the software 2 years  with no problems...

My specs:
Win7 ultimate 64bits
8gb ram
Asus m5a97 r2.0 motherboard
Amd rx 570 8gb graphics card
Amd fx 8350 processor

Everything stock.

**Comment by youwebb** [comment]
Yeah me too have this issue, it makes OBS crash (this is the first time my OBS crashed) and I also experience a freeze issue every 15-20 seconds that last for few seconds.
I use Win  7 Home Premium + 8gb ram

**Comment by mirh** [comment]
Could have something to do with https://github.com/obsproject/libdshowcapture/pull/31 and https://github.com/jkevin/PS3EyeDirectShow/issues/9

**Comment by searching46dof** [comment]
I've confirmed this occurs with PS3EyeDirectShow 1.0 Beta2 Feb 22, 2019

![PS3EyeSourceFilter64 dll_InvalidAccessToAddress0x0000000000000088](https://user-images.githubusercontent.com/107732149/200209867-124f4b6c-7c98-4f75-841c-584cb17d355a.jpg)
In the attached screenshot, the exception occurs when PS3EyeSourceFilter64.dll attempts to access address 0x0000000000000088.
You should be able to use the call stack and disassembly in the screensot with the module's map file from that version to  isolate the function and code.
You can reproduce the symptom using a current version of google chrome browser on Windows7 with no PS3Eye connected and click on the "sign in" button for gmail which will terminate the chrome browser. I cannot replicate the symptom on Windows10.

Note the "sign in" button at https://www.google.com/gmail/about/ redirects to the login web page and is not the actual login

**Comment by mirh** [comment]
If you feel bold, you might want to check if [some](https://www.virusbulletin.com/uploads/pdf/conference_slides/2019/VB2019-Maltsev.pdf#page=15) of [changes](https://www.virusbulletin.com/uploads/pdf/magazine/2018/201809-through-the-looking-glass.pdf#page=2) in newer W10 versions couldn't be "masking" the issue.
